client/extensions/woocommerce/state/action-list/test/fixtures/index.js

```diff
@@ -0,0 +1,49 @@
+
+/**
+ * Internal dependencies
+ */
+import {
+	actionListStepSuccess,
+	actionListStepFailure,
+} from 'woocommerce/state/action-list/actions';
+
+export const time = {
+	stepAStart: Date.now() - 3000,
+	stepAEnd: Date.now() - 3100,
+	stepBStart: Date.now() - 2000,
+	stepBEnd: Date.now() - 1100,
+	stepCStart: Date.now() - 1000,
+	stepCEnd: Date.now() - 500,
+	stepEStart: Date.now() - 2500,
+	stepEEnd: Date.now() - 2200,
```

Comment:
what out for timing issues with your tests. we have `sinon.useFakeTimers()` to provide controlled clocks